corehq/apps/translations/integrations/transifex/client.py

```diff
@@ -143,51 +158,61 @@ def get_translation(self, resource_slug, hq_lang_code, lock_resource):
         :param lock_resource: lock resource after pulling translation
         :return: list of POEntry objects
         """
-        language_id = self._lang_code_to_language_id(self.transifex_lang_code(hq_lang_code))
-        language = self.api.Language(id=language_id)
-        resource = self.api.Resource.get(slug=resource_slug, project=self.project)
-        download = self.api.ResourceTranslationsAsyncDownload.download(resource=resource, language=language)
-        response = requests.get(download, stream=True)
+        resource = self._get_resource(resource_slug)
+        language_id = self._to_language_id(self.transifex_lang_code(hq_lang_code))
+        content = self._download_resource_translations(resource.id, language_id)
         temp_file = tempfile.NamedTemporaryFile()
         with open(temp_file.name, 'w', encoding='utf-8') as f:
-            f.write(response.content.decode(encoding="utf-8"))
+            f.write(content.decode(encoding="utf-8"))
         if lock_resource:
             resource.save(accept_translations=False)
         return polib.pofile(temp_file.name)
 
-    @staticmethod
-    def _lang_code_to_language_id(lang_code):
-        return f"l:{lang_code}"
-
-    @staticmethod
-    def _language_id_to_lang_code(language_id):
-        return language_id.replace("l:", "")
-
-    @staticmethod
-    def transifex_lang_code(hq_lang_code):
-        """
-        Single place to convert lang codes from HQ to transifex lang code
+    def move_resource(self, old_resource_slug, new_resource_slug):
+        old_resource = self._get_resource(slug=old_resource_slug)
+        self._create_resource(new_resource_slug, old_resource.name)
+        self.delete_resource(old_resource_slug)
 
-        :param hq_lang_code: lang code on HQ
-        """
-        return SOURCE_LANGUAGE_MAPPING.get(hq_lang_code, hq_lang_code)
+    def get_project_langcodes(self):
+        language_stats_list = self._list_language_stats()
+        return [self._to_lang_code(stats.language.id) for stats in language_stats_list]
 
     def source_lang_is(self, hq_lang_code):
```

Comment:
would love if this was `is_source_lang`

Reply:
I think I named it such considering it returns a boolean, so when its used it sounds nice like
```
if source_lang_is("en"):
    # Do the following
```